RELEASE_PROCESS.rst

```diff
@@ -32,38 +56,60 @@ changed.
 
 ::
 
-     git diff HEAD~1
+    git diff HEAD~1
 
 
 Test the release
 ----------------
 
 ::
 
-     make check
+    make check
 
 
 Really test the release
 -----------------------
 
+Test it in the charm on juju 1.25...
+
+::
+
+    # Assuming $REPO is where your git repos live and $RELEASE is your release
+    # number:
```

Comment:
Sorry I didn't mention this earlier, but REPO must be a full path, not relative.  I always use $HOME/git to ensure